service/service.go

```diff
@@ -0,0 +1,310 @@
+package service
+
+import (
+	"io"
+	"net"
+	"os"
+	"sync"
+
+	"github.com/gnolang/gno/tm2/pkg/crypto/keys"
+	"github.com/gnolang/gnomobile/service/gnomobiletypes"
+	"go.uber.org/zap"
+	"google.golang.org/grpc"
+)
+
+const SOCKET_FILE = "gnomobile.sock"
+
+type GnomobileService interface {
+	gnomobiletypes.GnomobileServiceServer
+
+	GetSocketPath() string
+
+	io.Closer
+}
+
+type gnomobileService struct {
+	logger     *zap.Logger
+	client     *Client
+	rootDir    string
+	socketPath string
+	lock       sync.RWMutex
+
+	remote  string
+	chainID string
+
+	activeAccount keys.Info
```

Comment:
Why does the active account need to be maintained in this struct? It seems that this duplicates the functionality of nameOrBech32 in the Client struct.
https://github.com/gnolang/gnomobile/blob/a07f32725fef3ec12c256445659d00afb81ade2e/service/client.go#L23